src/mintpy/prep_hyp3.py

```diff
@@ -109,9 +109,13 @@ def add_hyp3_metadata(fname, meta, is_ifg=True):
     # HDF-EOS5 metadata
     if hyp3_meta['ReferenceGranule'].startswith('S1'):
         meta['beam_mode'] = 'IW'
+
+    meta['startUTC'] = date1.strftime('%Y-%m-%d %H:%M:%S.%f') + '+00:00'
+    meta['stopUTC'] = date2.strftime('%Y-%m-%d %H:%M:%S.%f') + '+00:00'
+    meta['relativeOrbit'] = ((int(hyp3_meta['ReferenceOrbitNumber']) - 73) % 175) + 1
```

Comment:
The relative orbit calculation is different for S1A and S1B, as shown in [`s1_reader`](https://github.com/isce-framework/s1-reader/blob/bb585c77ee538324212085f4737f3d2d534ac998/src/s1reader/s1_burst_slc.py#L1115-L1118).

Reply:
On it!

Reply:
Looks like the example HyP3 data is an older version that doesn't include some metadata fields. I'll create a new version.